packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -56,6 +57,9 @@ export class TableRow<
     @attr({ attribute: 'hide-selection', mode: 'boolean' })
     public hideSelection = false;
 
+    @attr({ mode: 'boolean' })
+    public expanded = false;
```

Comment:
Where does this get set to `true`?

Reply:
Ultimately, this will be set from the `Table` template, but I was going to leave that for the next PR I put together.